zetaclient/interfaces.go

```diff
@@ -48,8 +48,8 @@ type ChainSigner interface {
 
 // ZetaCoreBridger is the interface to interact with ZetaCore
 type ZetaCoreBridger interface {
-	PostSend(zetaGasLimit uint64, msg *crosschaintypes.MsgVoteOnObservedInboundTx) (string, string, error)
-	PostReceiveConfirmation(
+	PostVoteInbound(gasLimit, retryGasLimit uint64, msg *crosschaintypes.MsgVoteOnObservedInboundTx) (string, string, error)
```

Comment:
retryGasLimit - I see you're passing the execution gas limit here. Should the name be execution gas limit?